unfetter-discover/unfetter-ui
# Cli

fixes unfetter-discover/unfetter#446
requires PR, https://github.com/unfetter-discover/unfetter/pull/624

## To Test
Verify the following commands
```
cd unfetter-ui
rm -rf node_modules
npm install
npm run lint
npm run doc
npm run test
npm run coverage
npm run build:dev
npm run analyze:dev
npm run build:prod
npm run serve:local:dev:demo
```
Open the application with `npm run serve:local:dev:demo`, verify the pages the work.  Edit source code for a page, notice Hot Module Reloading updates page, but does not do a full refresh.
